### ISSUE TITLE
Attenuators / Silencers - Main, Cross Talk

### ISSUE THREAD

**Issue by Richard-Jackson-Arup** [theodi/BDNS#292]
**device/asset description**
Attenuators / Silencers - Ventilation Plant
Attenuators / Silencers - Cross Talk

**device/asset name**
ATT
CTA

I note that the current general abbreviation for is SLCR for silencer which does not allow easy separation of the key types and is also not really typical of my experience of language in UK market. Happy to be corrected but Silencer is more US language, ATT and Attenuator is more common in UK to my me.

**device/asset documentation**
Provide link to a datasheet/manual/any supporting documentation
[example document](https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf)

**Comment by RitaLav** [comment]
Adding the ventilation plan and cross talk would be ok - further discussion first about general abbreviation 'SLCR'

**Comment by RitaLav** [comment]
@Richard-Jackson-Arup 
Could the asset type help in this case, to differentiate the attenuator types? 
https://github.com/theodi/BDNS/blob/master/BDNS_Specification_naming_syntax.md#deviceasset-type-assettype

**Comment by Richard-Jackson-Arup** [comment]
I don't think that works very well as it is unclear to casual reader what the type referencing means in this case.

I can see me using maybe the type referencing to cover system type a lot if/when I want to include that and the fact that is slightly impenetrable is not so much of an issue because the person who needs to know what that means will.

I this case it is pretty basic differentiation to make the system easier and more coherent to use.

I think the terms ATT and CTA are also pretty widely used. Seems a strange one to not just do given the degree of granularity already provide for a lot of other items.

What about the SLCR vs ATT? I might ask around Mech teams beyond my own, see what people are doing in case it is just me being obtuse.

**Comment by jgunstone** [comment]
I asked our acoustics team and for them being able to use ATT instead of SLCR is very important. 
I also agree having CTA is useful

**Comment by jgunstone** [comment]
Hi - 
yes ATT not SLCR. 
and yes basically the same reasoning - I think whilst Silencer (SLCR) also makes sense it is scarcely used in a UK market and goes against the norms that they have established over many years of operation.

**Comment by RitaLav** [comment]
@pisuke @JimGriffin88 @TheFridgeShaman 

On the call today we discussed about the potential of replacing SLCR with ATT, and effectively deprecating SLCR.

We would need to agree on technical details on how to do so, but please let us know if you have any comments on the principle.

**Comment by JimGriffin88** [comment]
> @pisuke @JimGriffin88 @TheFridgeShaman
> 
> On the call today we discussed about the potential of replacing SLCR with ATT, and effectively deprecating SLCR.
> 
> We would need to agree on technical details on how to do so, but please let us know if you have any comments on the principle.

Hi Rita,

I'm in agreement with BDNS adopting the 'industry norm' abbreviations wherever possible and would believe ATT is an example of this from my limited exposure to Mech Vent design. 

Worth noting perhaps  that I am aware of at least 1 project utilising SLCR at the moment.

The logistics of deprecating the existing abbreviation sounds challenging but having a method in place for this is undoubtedly worth the time to define.
